activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -461,7 +462,6 @@ def initialize_type_map(m) # :nodoc:
           m.register_type 'ltree', OID::SpecializedString.new(:ltree)
 
           # FIXME: why are we keeping these types as strings?
-          m.alias_type 'interval', 'varchar'
```

Comment:
Unfortunately this is a breaking change, so we can't actually do this without a deprecation cycle. We should continue to treat it as a string and emit a warning, since w/ the attributes API you can now specify manually in the mean time.